.github/workflows/push.yml

```diff
@@ -0,0 +1,65 @@
+name: Unit tests & Build Testapp
+
+on: ['push', 'pull_request']
+
+jobs:
+
+  flake8:
+    name: Flake8 tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout python-for-android
+      uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1.1.0
+      with:
+        python-version: 3.7
+    - name: Run flake8
+      # we use exactly the same exceptions than tox.ini file
+      run: |
+        python -m pip install --upgrade pip
+        pip install -U flake8
+        flake8 --ignore=E123,E124,E126,E226,E402,E501,W503,W504 pythonforandroid/ tests/ ci/
```

Comment:
why not a `tox -e flake8`?

Reply:
Aaa...yes...that should work, honestly, I had a different idea about flake8 tests (and never finished that) ...let me explain...

With some `gh-actions`, we can interact with the web ui interface of github, so, I was thinking that it would be great that the detected PEP8 errors could be notified automatically at PRs ...mmm...better check this action to get a better idea: https://github.com/suo/flake8-github-action . So... I did not implement this because that flake8 action cannot be configured (at least for now, but it could be another options...needs investigation)

So, said this, **we could give it a try to the tox command (I will try this when I have some time)**, but I would keep one eye on this flake8 action mentioned above, because we could have some improvements here...specially thinking on PR reviewers...it seems that the `red cross` in travis tests, somehow and sometimes, is not seen, right? :laughing: